web/main/global.d.ts

```diff
@@ -17,3 +17,6 @@ interface Navigator {
       }
     | undefined
 }
+
+declare module '*.avif'
```

Comment:
Can I ask you to find smaller images on Wikipedia?

300 KB is too much.

For PNG, it is better to use some logotype, because photo is bad in PNG compression.

Also, you can use https://squoosh.app/ to convert it to AVIF or WebP to reduce the size.

I had a few projects where tests assets after 2 years of project history bloat `.git` size.